patches/0002_js_mse_eme_mvt_AudioTagChangeFor_FMP4_FMP3_stream.patch

```diff
@@ -0,0 +1,41 @@
+diff --git a/harness/main.js b/harness/main.js
+index c722460..5fde508 100644
+--- a/harness/main.js
++++ b/harness/main.js
+@@ -156,14 +156,14 @@ var createRunner = function(testSuite, testSuiteVer, testsMask) {
+   window.globalRunner = runner;
+ 
+   var id = 0;
+-  runner.getNewVideoTag = function() {
++  runner.getNewVideoTag = function(tag) {
+     var testarea = document.getElementById('testarea');
+     var vid = 'v' + id;
+     if (harnessConfig.recycleVideoTag)
+       ++id;
+     if (!document.getElementById(vid)) {
+       testarea.innerHTML = '';
+-      testarea.appendChild(util.createElement('video', vid, 'box-right'));
++      testarea.appendChild(util.createElement(tag, vid, 'box-right'));
+       document.getElementById(vid).controls = true;
+       document.getElementById(vid).muted = harnessConfig.muted;
+       if (harnessConfig.fullscreen) {
+diff --git a/harness/test.js b/harness/test.js
+index b781e31..262c06d 100644
+--- a/harness/test.js
++++ b/harness/test.js
+@@ -380,7 +380,14 @@ TestExecutor.prototype.startNextTest = function() {
+ 
+   this.startTime = Date.now();
+   this.currentTest.runner = this;
+-  this.currentTest.video = this.getNewVideoTag();
++  var tag='video';
++  if (this.currentTest.desc.includes("FMP4_MP3")) {
++     tag='audio';
++  }
++  else {
++     tag='video';
++  }
++  this.currentTest.video = this.getNewVideoTag(tag);
```

Comment:
Now function getNewVideoTag is related to both Audio/Video..Do we need to change the name.
like getNewAudioVideoTag()